alembic/testing/plugin/pytestplugin.py

```diff
@@ -50,6 +52,30 @@ def pytest_pycollect_makeitem(collector, name, obj):
         return []
 
 
+def vendored_exclusions(compound_object):
```

Comment:
**mike bayer** (zzzeek) wrote:

I've re-vendored this into alembic again in alembic/testing/exclusions.py so we can remove this